src/Page/News.elm

```diff
@@ -172,6 +161,15 @@ summaryFromArticleBody articleBody =
         |> String.join " "
 
 
+newsArticleImage : Maybe String -> Html msg
+newsArticleImage maybeImage =
+    let
+        imageSource =
+            Maybe.withDefault "/images/news/article_1.jpg" maybeImage
+    in
+    img [ src imageSource, css [ newsImageStyle ] ] []
```

Comment:
If we can't get alt text from PlaceCal - provide `alt ""` here so we are deliberately saying the image is eye-candy rather than essential info.